library/common/src/main/java/com/google/android/exoplayer2/util/TimestampAdjuster.java

```diff
@@ -33,7 +33,7 @@
    * The value one greater than the largest representable (33 bit) MPEG-2 TS 90 kHz clock
    * presentation timestamp.
    */
-  private static final long MAX_PTS_PLUS_ONE = 0x200000000L;
+  public static final long MAX_PTS_PLUS_ONE = 0x200000000L;
```

Comment:
Can keep this private and move `adjustCueForPtsWrap(long)` into this class, pretty much a personal preference choice I could go either way.